handel/package.json

```diff
@@ -18,7 +18,8 @@
     "ajv-errors": "^1.0.0",
     "archiver": "^1.3.0",
     "aws-sdk": "^2.169.0",
-    "fs-extra": "^4.0.2",
+    "common-tags": "^1.7.2",
```

Comment:
This package gives us some nice utilities around tagged template literals, which can help us have nice pretty error messages in the extensions API (since I think that may get confusing to people if we don't have friendly error messages).